<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>881</height>
    </rect>
   </property>
   <property name="title">
    <string>Best Camera Application</string>
   </property>
   <widget class="QLabel" name="labelPixmap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FIle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOption">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>1011</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Image</string>
    </property>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(171, 195, 198);</string>
     </property>
    </widget>
    <widget class="QLabel" name="ResizeImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Resize Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineMax">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineMin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineStep">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editStep">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMax">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHsv">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="Coloringmage">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Coloring Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnApply">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>200</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Spin Box</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="sliderRed">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderGreen">
     <property name="geometry">
      <rect>
       <x>341</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderBlue">
     <property name="geometry">
      <rect>
       <x>396</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hue</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderHue">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>179</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderSaturation">
     <property name="geometry">
      <rect>
       <x>584</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderValue">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>101</y>
       <width>41</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHsv_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>219</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="FilterImage">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>32</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Filtering Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThresh">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Thresholding</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>87</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBlur">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Blur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCanny">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Canny edge</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCamera">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecord">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rec Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
